everviz/plugins/_crossplot_plugin/_dash_components/_crossplot_callback.py

```diff
@@ -0,0 +1,17 @@
+def _crossplot_callback(
+    data, xaxis_column_name, yaxis_column_name, xaxis_type, yaxis_type
+):
+    return {
+        "data": data,
+        "layout": dict(
+            xaxis={
+                "title": xaxis_column_name,
+                "type": "log" if xaxis_type == "Log" else "linear",
```

Comment:
This is a fairly specific logic, why isn't it just `xaxis_type`?

If you need specific format, use a mapping?:
`_LOG_TYPE_MAPPING = {"Log": "log", "Linear": "linear}`

```
"title": xaxis_column_name,
"type": _LOG_TYPE_MAPPING[xaxis_type]
```

Reply:
Not opposed to this idea, however there is also a third option, `Cumulative`, which maps to `Linear`, of course no problem to add that to the map, but think that might make it harder to read.

Regarding your idea @DanSava, sure, we could just make it lowercase up front, but like the case is now, there are three options, Linear, Log and Cumulative, where only two of them can be used as axis_type. Could add some pre-processing and be more flexible, but would we need to?

Reply:
I thought all of them `Linear, Log and Cumulative`  were possible axis_types when I suggested my idea...